src/app/static/src/app/components/ResetConfirmation.vue

```diff
@@ -79,6 +80,13 @@
             })
         },
         methods: {
+            toggleFocus: function (element: HTMLElement, toggle: boolean) {
+                if (element){
+                    this.$nextTick(() => {
+                        toggle ? element.focus() : element.blur();
```

Comment:
The blur in this case is perhaps redundant as the element is removed from the DOM when it closes.

Reply:
This isn't an expression, so using the ternary operator is a bit confusing - a plain if/else would be clearer.